packages/nc-gui-v2/components/smartsheet/sidebar/MenuBottom.vue

```diff
@@ -94,6 +96,7 @@ function onOpenModal(type: ViewTypes, title = '') {
       </a-tooltip>
     </a-menu-item>
 
+    <SmartsheetSidebarMenuApiSnippet :model-value="showApiSnippet" @update:model-value="(val) => (showApiSnippet = val)" />
```

Comment:
why not just use `v-model="showApiSnippet"`?
It does the exact same as `:model-value="value" @update:model-value="value = $event"`